source/_components/device_tracker.linksys_ap.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: "Linksys Access Points"
+description: "Instructions how to integrate Linksys Access Points into Home Assistant."
+date: 2016-12-16 01:40
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Presence Detection
+featured: false
+ha_release: 0.35
+---
+
+The `linksys_ap` platform offers presence detection by looking at connected devices to a Linksys based access point.
+It was tested with a LAPAC1750 AC1750 Dual Band Access Point.
+
+To use a Linksys Access Point in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: linksys_ap
+    host: 192.168.1.1
+    username: admin
+    password: password
+    verify_ssl: true
```

Comment:
Can you please remove the optional configuration entries to keep this sample minimal?